### ISSUE TITLE
Pluginable Linter

### ISSUE THREAD

**Issue by sterpe** [deedubs/es6-plato#2]
Any chance to have the linter as a pluginable piece or switch to eslint?

**Comment by ClemMakesApps** [comment]
Yeah, that would be quite helpful. I think it is currently using JSHint (as per original Plato)? A lot of my ES6 code is giving lint errors but other than that, this es6-plato is great!
